content/posts/GSoC_2021_Final/index.md

```diff
@@ -0,0 +1,157 @@
+
+> Note: Only the parent's APIs are used in some backends, so for each of the individual public functions like `load_glyph`, `load_char`, `get_kerning`, etc., we find the FT2Font object which has that glyph from the parent FT2Font cache!
+
+### Multi-Font embedding in PDF/PS/EPS
+Now that we have multiple fonts to render a string, we also need to embed them for those special backends (i.e., PDF/PS, etc.). This was done with some patches to specific backends:
+- [PR: Implement multi-font embedding for PDF Backend](https://github.com/matplotlib/matplotlib/pull/20804)
+- [PR: Implement multi-font embedding for PS Backend](https://github.com/matplotlib/matplotlib/pull/20832)
+
+With this, one could create a PDF or a PS/EPS document with multiple fonts which are embedded (and subsetted!).
+
+## Conclusion
+From small contributions to eventually working on a core module of such a huge library, the road was not what I had imagined, and I learnt a lot while designing solutions to these problems.
+
+#### The work I did would eventually end up affecting every single Matplotlib user.
+...since all plots will work their way through the new codepath!
+
+I think that single statement is worth the <ins>whole GSoC project</ins>.
+
+For the sake of statistics (and to make GSoC sound a bit less intimidating), here's a list of contributions I made to Matplotlib before Summer '21, most of which are only a few lines of diff:
+
+|              |                                                                                                                         |                 |        |
+|--------------|-------------------------------------------------------------------------------------------------------------------------|-----------------|--------|
+| Nov 2, 2020  | [Expand ScalarMappable.set_array to accept array-like inputs](https://github.com/matplotlib/matplotlib/pull/18870)      | (+28 −4)        | MERGED |
+| Nov 8, 2020  | [Add overset and underset support for mathtext](https://github.com/matplotlib/matplotlib/pull/18916)                    | (+71 −0)        | MERGED |
+| Nov 14, 2020 | [Strictly increasing check with test coverage for streamplot grid](https://github.com/matplotlib/matplotlib/pull/18947) | (+54 −2)        | MERGED |
+| Jan 11, 2021 | [WIP: Add support to edit subplot configurations via textbox](https://github.com/matplotlib/matplotlib/pull/19271)      | (+51 −11)       | DRAFT  |
+| Jan 18, 2021 | [Fix over/under mathtext symbols](https://github.com/matplotlib/matplotlib/pull/19314)                                  | (+7,459 −4,169) | MERGED |
+| Feb 11, 2021 | [Add overset/underset whatsnew entry](https://github.com/matplotlib/matplotlib/pull/19497)                              | (+28 −17)       | MERGED |
+| May 15, 2021 | [Warn user when mathtext font is used for ticks](https://github.com/matplotlib/matplotlib/pull/20235)                   | (+28 −0)        | MERGED |
+

```

Comment:
Can you add a small bullet list summarizing the GSOC 2021 PRs? As a done/in progress/to do?